Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -1161,6 +1163,9 @@ extension SourceKitLSPServer {
     if let diagnosticOptions = server.diagnosticProvider {
       await registry.registerDiagnosticIfNeeded(options: diagnosticOptions, for: languages, server: self)
     }
+    if let codeLensOptions = server.codeLensProvider {
+      await registry.registerCodeLensIfNeeded(options: codeLensOptions, for: languages, server: self)
+    }
```

Comment:
You should also add this to the default `SourceKitLSPServer.serverCapabilities`. See comment at top of this function.